Traceback (most recent call last):
  File "/u/sdxpy/builder/pattern_attempt.py", line 32, in <module>
    builder.build()
  File "/u/sdxpy/builder/pattern_attempt.py", line 8, in build
    self.extract_rules()
  File "/u/sdxpy/builder/pattern_attempt.py", line 19, in extract_rules
    self.rules[target] = self.graph.nodes[target]["recipes"]
KeyError: 'recipes'
